realm/realm-library/src/main/java/io/realm/internal/TableView.java

```diff
@@ -826,6 +832,7 @@ public long sync() {
     private native long nativeFindAllFloat(long nativePtr, long columnIndex, float value);
     private native long nativeFindAllDouble(long nativePtr, long columnIndex, double value);
     private native long nativeFindAllDate(long nativePtr, long columnIndex, long dateTimeValue);
+    private native long nativeFindBySourceNdx(long rowIndex);
```

Comment:
This doesn't match the native function which takes two params, tableview ptr and the rowIndex. This is the reason of native crash.